packages/keystone/test.utils.js

```diff
@@ -628,23 +628,28 @@ const expectToThrowGQLError = async (testFunc, errorFields, path = 'obj') => {
         if (!interpolatedMessageForUser) throw new Error(`expectToThrowGQLError(): you need to set ${errorFields.messageForUser} for locale=${locale}`)
     }
     const message = template(errorFields.message)(errorFields.messageInterpolation)
+    // NOTE: In case where only type and code provided message should not be checked
+    const messageFields = message ? { message } : {}
+    // NOTE: In case where is no errorFields.messageForUser interpolatedMessageForUser becomes undefined,
+    // so we should not check it
+    const messageForUserFields = interpolatedMessageForUser ? { messageForUser: interpolatedMessageForUser } : {}
```

Comment:
Above there is an exception for this case

Reply:
There's not. Exception is thrown only if `messageForUser` if branch is triggered, which is not the case when we're expecting only type + code.

Reply:
Oops, didn't notice that